plugins/inventory/cobbler.py

```diff
@@ -177,6 +173,17 @@ def _add_safe_group_name(self, group, child=None):
             self.inventory.add_child(group_name, child)
         return group_name
 
+    def _exclude_profile(self, profile):
+        if len(self.include_profiles) > 0:
```

Comment:
More Pythonic would be:
```suggestion
        if self.include_profiles:
```